### DISCUSSION TITLE
Automate new static deploy on content change

### DISCUSSION THREAD

**Discussion by Sander-Riel** [vercel/next.js#12273] [Help]
I have set up a next site with a CAAS that supports webhooks (prismic) and I'm trying to set up a system where I create a new static build and upload that to my website every time my content changes.

My git repository is stored on Gitlab. Has anybody done that before and would that person care to share how it's done?

Or give me a pointer in the right direction?

**Comment by stigkj** [comment]
Not sure if this would work for you, but there is an RFC coming up (Incremental Static Generation): https://github.com/zeit/next.js/discussions/11552
